### PR TITLE
[DSD-5479] add database specific to different use case

### PR DESCRIPTION
**Pull request by challabeehyv**

### CODE REVIEW

File: certify-default.properties [hunk 1]
```diff
@@ -144,3 +144,4 @@ mosip.certify.cache.expire-in-seconds={'userinfo': ${mosip.certify.access-token-
 
 mosip.certify.domain.url=
 mosip.certify.key-values={}
+mosip.certify.database.name=
```
Comment (vishwa-vyom) [inline certify-default.properties:147]: we can give a default value like mosip_certify for this

File: certify-mock-identity.properties [hunk 2]
```diff
@@ -13,6 +13,7 @@ mosip.certify.cache.store.individual-id=true
 mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
 mosip.certify.mock.vciplugin.verification-method=${mosip.certify.authn.jwk-set-uri}
 mosip.certify.mock.authenticator.get-identity-url=https://${mosip.api.public.host}/v1/mock-identity-system/identity
+mosip.certify.database.name=mosip_certify_mock
```
Comment (vishwa-vyom) [inline certify-mock-identity.properties:16]: it has to be in the "Use specific default overriding properties section" we can put it above mosip.certify.key-values

File: certify-mock-identity.properties [hunk 3]
```diff
@@ -13,6 +13,7 @@ mosip.certify.cache.store.individual-id=true
...
 
 ## ------------------------------------------- Use specific default overriding properties ------------------------------------------------------------
```
Comment (vishwa-vyom) [inline certify-mock-identity.properties:18]: lets correct it to say Usecase and not use

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: certify-default.properties (hunk 1 above)
Comment (challabeehyv) [reply certify-default.properties]: added this

File: certify-mock-identity.properties (hunk 2 above)
Comment (challabeehyv) [reply certify-mock-identity.properties]: moved to use case specific  properties

File: certify-mock-identity.properties (hunk 3 above)
Comment (challabeehyv) [reply certify-mock-identity.properties]: changed

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**